diagnostic.md

```diff
@@ -45,7 +45,7 @@ while (x < 5) {
 
 How many times does the loop run?  What's the value of `x` after the while loop?
 
-_Replace this text with your answer._
+_the loop will run 4 times and the value of x after the loop will equal 5_
```

Comment:
Close, but the loop will run 3 times and x will be a value of '8'. Reason being:
Since x = 1, first time through the loop, the value of x will be 2.
-Since 2 < 5, it will run again making the value of x = 4
- Since 4 < 5, it will run again making the value of x = 8
- But since 8 not less than 5, it will not run again and exit the loop.

Thus making it run 3 times and value of x = 8.